### ISSUE TITLE
IPL: ReadTimeout following old PNOR update

### ISSUE THREAD

**Issue by cdeadmin** [openbmc/openbmc#2271]
```
$ ./rest_cmd -i $BMC_IP -o GET -u /org/openbmc/control/flash/bios
Executing: GET  /org/openbmc/control/flash/bios

/org/openbmc/control/flash/bios
{
    "description": "Internal Server Error",
    "exception": "DBusException('Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken.',)",
    "traceback": [
        "Traceback (most recent call last):",
        "  File \"/usr/lib/python2.7/site-packages/bottle.py\", line 862, in _handle",
        "    return route.call(**args)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 799, in __call__",
        "    return self.callback(*a, **kw)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 856, in wrap",
        "    return callback(*a, **kw)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 1039, in __call__",
        "    return self.callback(*a, **kw)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 1013, in wrap",
        "    return self.to_jsonp(callback(*a, **kw))",
        "  File \"/usr/lib/python2.7/site-packages/bottle.py\", line 1734, in wrapper",
        "    rv = callback(*a, **ka)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 978, in wrap",
        "    data = callback(*a, **kw)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 200, in __call__",
        "    return getattr(self, 'do_' + request.method.lower())(**kw)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/wsgi/apps/rest_dbus.py\", line 538, in do_get",
        "    mapper_data=request.route_data['map'])",
        "  File \"/usr/lib/python2.7/site-packages/obmc/mapper/bindings.py\", line 127, in enumerate_object",
        "    path, owner, interfaces, match))",
        "  File \"/usr/lib/python2.7/site-packages/obmc/mapper/bindings.py\", line 111, in __get_properties_on_bus",
        "    properties_iface, i))",
        "  File \"/usr/lib/python2.7/site-packages/obmc/mapper/bindings.py\", line 97, in __get_properties_on_iface",
        "    properties_iface.GetAll, iface)",
        "  File \"/usr/lib/python2.7/site-packages/obmc/mapper/bindings.py\", line 83, in __try_properties_interface",
        "    return f(*a)",
        "  File \"/usr/lib/python2.7/site-packages/dbus/proxies.py\", line 145, in __call__",
        "    **keywords)",
        "  File \"/usr/lib/python2.7/site-packages/dbus/connection.py\", line 651, in call_blocking",
        "    message, timeout)",
        "DBusException: org.freedesktop.DBus.Error.NoReply: Did not receive a reply. Possible causes include: the remote application did not send a reply, the message bus security policy blocked the reply, the reply timeout expired, or the network connection was broken."
    ]
}
```

<cde:info> SW401399 </cde:info>

**Comment by anoo1** [comment]
Commit from last month changed the path for the executable files:
https://github.com/openbmc/openbmc/commit/0ad3b1df9079064491c8318bece22e63cc71b2f4

The bios flasher was still expecting the previous path, and therefore not starting correctly.

**Comment by rfrandse** [comment]
<https://gerrit.openbmc-project.xyz/6541> flashbios: Do not expect the full path to the executable
Resolves  #2271 IPL: ReadTimeout following old PNOR update
